diff -Nur ganeti-1.2~b3/man/ganeti-os-interface.in ganeti-1.2~b3.new/man/ganeti-os-interface.in
--- ganeti-1.2~b3/man/ganeti-os-interface.in	2007-11-26 16:24:18.000000000 +0000
+++ ganeti-1.2~b3.new/man/ganeti-os-interface.in	2007-11-28 18:18:04.914751868 +0000
@@ -3,10 +3,10 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "GANETI-OS-INTERFACE" "7" "26 November 2007" "ganeti 1.2" ""
+.TH "GANETI-OS-INTERFACE" "7" "28 November 2007" "ganeti 1.2" ""
 
 .SH NAME
-ganeti guest OS interface \- specifications for guest OS types
+ganeti-os-interface \- specifications for guest OS types
 .SH "DESCRIPTION"
 .PP
 The method of supporting guest operating systems in Ganeti is to
diff -Nur ganeti-1.2~b3/man/ganeti-os-interface.sgml ganeti-1.2~b3.new/man/ganeti-os-interface.sgml
--- ganeti-1.2~b3/man/ganeti-os-interface.sgml	2007-09-18 14:49:19.000000000 +0100
+++ ganeti-1.2~b3.new/man/ganeti-os-interface.sgml	2007-11-28 18:16:14.578781173 +0000
@@ -31,7 +31,7 @@
     <refmiscinfo>ganeti 1.2</refmiscinfo>
   </refmeta>
   <refnamediv>
-    <refname>ganeti guest OS interface</refname>
+    <refname>ganeti-os-interface</refname>
 
     <refpurpose>specifications for guest OS types
     </refpurpose>
