diff -urN ganeti-1.2.1.old/doc/examples/ganeti.initd.in ganeti-1.2.1/doc/examples/ganeti.initd.in
--- ganeti-1.2.1.old/doc/examples/ganeti.initd.in	2008-01-16 16:46:44.000000000 +0100
+++ ganeti-1.2.1/doc/examples/ganeti.initd.in	2008-01-18 14:56:45.723900500 +0100
@@ -6,7 +6,7 @@
 # Required-Start:    $syslog $remote_fs xend
 # Required-Stop:     $syslog $remote_fs xend
 # Default-Start:     2 3 4 5
-# Default-Stop:      S 0 1 6
+# Default-Stop:      0 1 6
 # Short-Description: Ganeti Xen Cluster Manager
 # Description:       Ganeti Xen Cluster Manager 
 ### END INIT INFO
