diff -urN ganeti-1.2.3.old/doc/examples/ganeti.cron ganeti-1.2.3/doc/examples/ganeti.cron
--- ganeti-1.2.3.old/doc/examples/ganeti.cron	2008-02-18 14:24:47.000000000 +0100
+++ ganeti-1.2.3/doc/examples/ganeti.cron	2008-02-22 08:23:41.911287638 +0100
@@ -1,3 +1,3 @@
 PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin
 # restart failed instances
-*/5 * * * * root /usr/local/sbin/ganeti-watcher
+*/5 * * * * root [ -x /usr/sbin/ganeti-watcher ] && /usr/sbin/ganeti-watcher
