Description: fix wrong docbook output
Author: Iustin Pop <iustin@debian.org>
Bug: http://code.google.com/p/ganeti/issues/detail?id=87
Last-Update: 2010-02-09
--- a/man/gnt-cluster.8.in
+++ b/man/gnt-cluster.8.in
@@ -360,7 +360,7 @@
 Note that only active disks can be checked by this command; in
 case a disk cannot be activated it's advised to use
 \fBgnt-instance activate-disks --ignore-size
-...\fR to force activation without regard to the
+\&...\fR to force activation without regard to the
 current size.
 .PP
 When the all disk sizes are consistent, the command will
